### PR TITLE
BASIQ-65: Remove Redux and use Context

### PR DESCRIPTION
**Pull request by albatahmet**
First context created and transaction invocation moved to context side
work will continue...

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1 +0,0 @@
-BASIQ_API_KEY=
```
Comment (k-cem-fd) [inline .env.sample]: This file is a sample to create new env file and put api key into it. Please re-add @albatahmet

File: components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
   const [isResumeModalOpen, openResumeModal, closeResumeModal] = useTernaryState(false);
   const [progressBarValue, setProgressBarValue] = useState(0);
   const [progressInterval, setProgressInterval] = useState(null);
-  const [transactionLoadingDispatched, setTransactionLoadingDispatched] = useState(false);
-
-  useDispatch(userTransactionsLoading());
-  const { isCompleted, transactionsError } = useSelector(state => state.userTransactions);
+  const transactionContext = useTransactionsDataContext();
+  const isCompleted = transactionContext.state.isCompleted;
+  const transactionsError = transactionContext.state.transactionsError;
```
Comment (k-cem-fd) [inline components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js]: Can you update as below or similar ? 
_const { isCompleted, transactionsError } = transactionContext.state;_

File: store/context/logger/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+			console.log('Prev state: ', prevState.current)
+			console.log('Next state: ', state)
+			console.groupEnd()
+		}
+		prevState.current = state
+	}, [state])
+
+	return [state, dispatchWithLogger]
+}
+
```
Comment (k-cem-fd) [inline store/context/logger/index.js:29]: Can you tidy this code using linter ?

File: store/context/transactionContext.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
+///Data
+import React, { createContext, useContext } from 'react';
+import useReducerWithLogger from './logger';
+import axios from 'axios';
+
+export const TransactionsDataContext = createContext();
+export const useTransactionsDataContext = () => useContext(TransactionsDataContext);
+
+///gerekli ilk stateler buraya yazılacak
```
Comment (k-cem-fd) [inline store/context/transactionContext.js]: Remove these comments or write in English if they are necessary

Comment (albatahmet) [reply store/context/transactionContext.js]: I removed all comment lines

File: .env.sample (hunk 1 above)
Comment (albatahmet) [reply .env.sample]: I re-added this file

File: components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js (hunk 2 above)
Comment (albatahmet) [reply components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js]: updated to a similar structure

File: store/context/logger/index.js (hunk 3 above)
Comment (albatahmet) [reply store/context/logger/index.js:29]: I edited it the way you said, is it correct?

Comment (k-cem-fd) [reply store/context/logger/index.js:29]: Yeah, perfect 👍

**Review by k-cem-fd — APPROVED** [review]

**Pull request closed without merging**